config/crds/tenancy.kcp.dev_clusterworkspaces.yaml

```diff
@@ -218,10 +218,10 @@ spec:
                   state are gated via the RBAC clusterworkspaces/initialize resource
                   permission."
                 items:
-                  description: ClusterWorkspaceInitializer is a unique string corresponding
+                  description: "ClusterWorkspaceInitializer is a unique string corresponding
                     to a cluster workspace initialization controller for the given
-                    type of workspaces.
-                  pattern: ^root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*(:[a-z][a-z0-9]([-a-z0-9]*[a-z0-9])?)$
+                    type of workspaces. \n TODO: Add tests for this regex"
```

Comment:
Do we want this TODO in user-facing doc?

Reply:
Nope 😄